src/util/validation.ts

```diff
@@ -0,0 +1,26 @@
+const isValidLength = (number: string, maxLength: number) => {
+    if (number.length < maxLength) {
+        return false;
+    } return true;
+}
+
+const isValidNumber = (number: string) => {
+    const regex = /^[0-9]*$/;
+    if (regex.test(number)) {
+        return true;
+    } return false;
+}
+
+const isValidMonthRange = (number: string) => {
+    if (Number(number)<=12 && Number(number)>=1) {
```

Comment:
소소하지만 아래처럼 작성하면 범위가 눈에 더 잘 들어올거 같네요(개인 취향이라 반영 안 해주셔도 됩니다)

```suggestion
    if (1 <= Number(number) && Number(number)<= 12 ) {
```

Reply:
확실히 최소, 최대 순으로 적는게 눈에 더 잘 들어오겠네요! 반영하겠습니다!